### ISSUE TITLE
Use env variables instead of config class

### ISSUE THREAD

**Issue by olivierobert** [shivabhusal/lacerate#19]
It's best practive to move all config variables to environment variables.

Using `figaro`, set up the file `application.yml` and move all config variables to it. Then use `Env.fetch('CONFIG_KEY')` to use it.

**Comment by shivabhusal** [comment]
Yes I have been using `figaro` since the beginning. Since there is no secret except facebook creds; only used in `devise.rb`
